.github/workflows/image-push-master.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         with:
           context: .
           push: true
+          platforms: linux/amd64,linux/arm64,linux/ppc64le
```

Comment:
According to https://github.com/docker/build-push-action/tree/releases/v2?tab=readme-ov-file it's recommended to setup [setup-buildx](https://github.com/docker/setup-buildx-action) action

Reply:
thanks @e0ne , Updated.